blocks/adp-collection-header/adp-collection-header.js

```diff
@@ -28,6 +37,16 @@ function createCollectionInfoHeader(collectionInfoHeader, collection) {
   const backButton = collectionInfoHeader.querySelector('.back-button a');
   backButton.href = createLinkHref(backButton.href);
 
+  document.getElementById('select-all-checkbox').addEventListener('click', function(event) {
+      // Get the checked state of the select all checkbox
+      const isChecked = event.target.checked;
+
+      // Get all the checkboxes within the cards
+      const checkboxes = document.querySelectorAll('.checkbox-container input[type="checkbox"], .filetype-video .checkbox-container input[type="checkbox"]');
+      // isChecked then selectAllAssets() else deselectAllAssets
+      isChecked ? selectAllAssets() : deselectAllAssets();
+  });
```

Comment:
This looks like it can be shortened, and should use an arrow function like the other event handlers.

```
  document.getElementById('select-all-checkbox').addEventListener('click', (event) => {
      event.target.checked ? selectAllAssets() : deselectAllAssets();
  });
  ```

Reply:
Done